.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+name: Build
+on:
+  pull_request:
+  push:
+    branches: [main] # Avoid duplicate builds on PRs.
+permissions:
+  contents: read
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [11, 8]
```

Comment:
Should we add `17` here too? (latest LTS) - wasn't in the original file so nothing we need to fix here, just an observation.